lib/buffer.js

```diff
@@ -624,10 +624,31 @@ function stringSlice(buf, encoding, start, end) {
   throw new ERR_UNKNOWN_ENCODING(encoding);
 }
 
-Buffer.prototype.copy =
-  function copy(target, targetStart, sourceStart, sourceEnd) {
-    return _copy(this, target, targetStart, sourceStart, sourceEnd);
-  };
+function validateOffsetType(val, name) {
+  if (val) {
+    if (typeof val !== 'number') {
+      throw new ERR_INVALID_ARG_TYPE(
```

Comment:
I think this can be all on one line as well?

Reply:
Ack